wpcom-legacy-redirector.php

```diff
@@ -5,6 +5,7 @@
  * Description: Simple plugin for handling legacy redirects in a scalable manner.
  * Version: 1.4.0-alpha
  * Requires PHP: 7.4
+ * Requires at least: 6.1
```

Comment:
Let's avoid the min supported WP version bump here - since the WP core `get_page_by_path()` function had caching added in WP 4.6.0, and the plugin development branch already requires WP 5.9, we don't need to make the change (also, this was never a complete version bump anyway, since other instances would have need to have been updated, such as docs and GitHub workflows).

```suggestion
```